src/components/SlateEditor/plugins/SlateToolbar/ToolbarButton.jsx

```diff
@@ -35,25 +36,26 @@ import { toolbarClasses } from './SlateToolbar';
 // @ndla/ui icon for Link type in toolbar has the same name as a link/anchor element component.
 // Thus triggering a false positive, that we have to disable.
 /* eslint-disable jsx-a11y/anchor-is-valid */
+const options = { ctrl: toKeyName('ctrl'), alt: toKeyName('alt') };
```

Comment:
Bruker is-hotkey sin oversettelse av navn på taster. På linux blir dette control og alt.